src/test/system_tests/ceph_s3_tests/test_ceph_s3_constants.js

```diff
@@ -1,40 +1,58 @@
 /* Copyright (C) 2022 NooBaa */
 "use strict";
 
+const cephalt_name = 'cephalt';
+const cephtenant_name = 'cephtenant';
+
+// NC ceph tests paths for bucket creation
+const FS_ROOT_1 = '/tmp/nsfs_root1/';
+const FS_ROOT_2 = '/tmp/nsfs_root2/';
+
 const CEPH_TEST = {
     test_dir: 'src/test/system_tests/ceph_s3_tests/',
     s3_test_dir: 's3-tests/',
     ceph_config: 'test_ceph_s3_config.conf',
     tox_config: 'tox.ini',
     new_account_params: {
-        name: 'cephalt',
+        name: cephalt_name,
         email: 'ceph.alt@noobaa.com',
```

Comment:
You can reuse the new variable `cephalt_name` in the email property.

Reply:
hmm I think you have an old version of the code, it's already there

Reply:
I meant:
```suggestion
        email: `${cephalt_name}@noobaa.com`,
```
but now I noticed that it is not the same... leave it as is.